tasks/util.py

```diff
@@ -1665,6 +1669,23 @@ def check_null_columns(self):
             raise ValueError('The following columns of the table "{table}" contain only NULL values: {columns}'.format(
                 table=self.output().table, columns=', '.join([x[0] for x in result])))
 
+    def check_universe_in_aggregations(self):
```

Comment:
Now that I see test changes, why don't we try to add a test for this check?

Reply:
Added test forcing the `ValueError` to raise when there is a `median` or an `average` aggregation without `universe` denominator